sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessSessionMessageEventArgs.cs

```diff
@@ -208,7 +208,7 @@ await _sessionReceiver.AbandonAsync(message, propertiesToModify, cancellationTok
         /// A lock token can be found in <see cref="ServiceBusReceivedMessage.LockToken"/>,
         /// only when <see cref="ReceiveMode"/> is set to <see cref="ReceiveMode.PeekLock"/>.
         /// In order to receive this message again in the future, you will need to save the <see cref="ServiceBusReceivedMessage.SequenceNumber"/>
-        /// and receive it using <see cref="ServiceBusReceiver.ReceiveDeferredMessageAsync(long, CancellationToken)"/>.
+        /// and receive it using <see cref="ServiceBusReceiver.ReceiveDeferredMessageAsync(long, System.Threading.CancellationToken)"/>.
```

Comment:
I'm curious where you are seeing this warning. I'm not seeing the warning in VS, and the [docs](https://docs.microsoft.com/en-us/dotnet/api/azure.messaging.servicebus.processsessionmessageeventargs.deferasync?view=azure-dotnet-preview#Azure_Messaging_ServiceBus_ProcessSessionMessageEventArgs_DeferAsync_Azure_Messaging_ServiceBus_ServiceBusReceivedMessage_System_Collections_Generic_IDictionary_System_String_System_Object__System_Threading_CancellationToken_) look to be generated correctly.

Reply:
I confess I mostly use Jetbrains Rider for development because I love it. I was assuming they are suppressed but Rider is still showing them so I went ahead and fixed them. When running in VS2019 latest it is only Resharper in there that gives me those warnings and not VS itself. Happy to close